src/main/java/net/kaczmarzyk/spring/data/jpa/utils/JsonBodyParams.java

```diff
@@ -43,9 +43,6 @@ public Collection<String> getParamValues(String paramKey) {
 	}
 
 	private static JsonElement getElementByKey(JsonElement rootElement, String key) {
-		if (rootElement == null){
-			return JsonNull.INSTANCE;
-		}
```

Comment:
why did you removed it from production code when the scope was just additional testing?